PixelHunter1995/GameStates/Playing.cs

```diff
@@ -21,15 +21,15 @@ public void Draw(GraphicsDeviceManager graphics, SpriteBatch spriteBatch, GameTi
             scene.Draw(graphics, spriteBatch, 1);
         }
 
-        public void Update(GameTime gameTime, Scene scene)
+        public void Update(GameTime gameTime, Scene scene, Input input)
```

Comment:
I think that we should possibly allow different input mappings depending on what state we're in but I really have no clue whether your code allows for this or not.

Reply:
When I asked in discord the general consensus seemed to be that we shouldn't, so I removed that functionality (a trace remains in the form of the parsers 'context' dictionary (with a single item), which I am unsure if I should remove (unused) or not (in case we change our mind)).

The input being passed to the function is the one in Main, which is currently the only Input instance, and is passed as I prefer that over static references or forcing everything to remember it in a field.